mobify/platform-scaffold
# WEB-1533: sfcc alt tag audit

This PR adds missing alt tags to images on the site. It resolves images included directly in the project and images parsed from Merlins. The SFCC audit will be in a separate PR. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1533
 **Linked PRs**: n/a

## Changes
- Added alt tags to banners on the homepage

## Todos

## How to test-drive this PR
- Run `npm start`
- Preview the [SFCC site](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- navigate through the site and check that no images are included without alt tags
